src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -30,23 +30,75 @@
+                                    <div class="dropdown-menu clickable">
+                                        @if (comment.Username == User.GetGitHubLogin())
+                                        {
                                             <form data-post-update="comments" asp-controller="Comments" asp-action="Delete" method="post" asp-route-reviewId="@Model.ReviewId">
-                                                <input type="hidden" name="commentId" value="@comment.CommentId"/>
+                                                <input type="hidden" name="commentId" value="@comment.CommentId" />
                                                 <button id="@comment.CommentId" type="submit" class="text-danger dropdown-item">Delete</button>
                                             </form>
                                             <a class="dropdown-item js-edit-comment">Edit</a>
+                                        }
+                                        <button type="button" class="btn dropdown-toggle dropdown-item" data-toggle="dropdown" aria-haspopup="true"
+                                                aria-expanded="false">
+                                            Create GitHub issue
+                                        </button>
+                                        <div class="dropdown-menu">
+                                            <table>
+                                                <tr>
+                                                    <td>
+                                                        <label class="btn btn-link">
+                                                            <a class="dropdown-item js-github" data-repo="azure-sdk-for-net"
+                                                               data-repo-language="c#">.NET</a>
+                                                        </label>
+                                                    </td>
+                                                    <td>
+                                                        <label class="btn btn-link">
+                                                            <a class="dropdown-item js-github" data-repo="azure-sdk-for-python"
+                                                               data-repo-language="python">Python</a>
+                                                        </label>
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td>
+                                                        <label class="btn btn-link">
+                                                            <a class="dropdown-item js-github" data-repo="azure-sdk-for-java"
+                                                               data-repo-language="java">Java</a>
```

Comment:
Why do we need data-repo-language here if we have it on review?

Reply:
We don't currently have it for Go on the review.

Reply:
Why do we need it for Java?

Reply:
I had it for consistency. I think this attribute can be removed once all languages have support for the review language.